src/main/java/sirius/web/controller/Controller.java

```diff
@@ -51,4 +51,14 @@
      * @param error the error which occurred
      */
     void onError(WebContext ctx, HandledException error);
+
+    /**
+     * In case processing a JSON request via a method fails (throws an exception), this method will be called.
+     * <p>
+     * This provides a convenient way to generate a "fallback" JSON response.
+     *
+     * @param ctx the context conatining the request
+     * @param error the error which occurred
+     */
+    void onJsonError(WebContext ctx, HandledException error);
```

Comment:
Muss das jetzt in jedem Controller überschrieben werden??

Und generell fänd ichs gut statt `ctx` als Variablenname `webContext` zu nutzen.

Reply:
Dein Parameternamen-Anliegen hab ich umgesetzt.

Das muss eigentlich nur in den Controllern angepasst werden, in denen eine spezielle JSON-Fehlerhandling-Logik benötigt wird. Alternativ könnte ich glaube auch das auch als abstrakte Klasse umbauen und die Standardlogik dort so einbauen, wie sie vorher in sirius.web.controller.ControllerDispatcher#handleFailure implementiert war.